src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -381,12 +394,11 @@ func applyClientConfig(client *client.Client, cfg *config.Config) {
 
 	// We offload this onto a go routine because creating the http clients
 	// blocks on waiting for the local server, and the local server starts
-	// on the thread, otherwise creating a deadlock.
+	// later on this same thread, so it would otherwise creating a deadlock.
 	go func() {
-		withHttpClient(cfg.Addr, config.Configure)
-		withHttpClient(cfg.Addr, geolookup.Configure)
 		withHttpClient(cfg.Addr, statserver.Configure)
```

Comment:
`statserver` uses the `HttpClient` to geolocate, we can also remove `Configure` here and reply on `pubsub` to keep consistency.